cmd/explaintest/t/explain_indexmerge.test

```diff
@@ -13,3 +13,5 @@ explain select * from t where b < 50 or c < 5000000;
 explain select * from t where a < 50 or b < 50 or c < 50;
 explain select * from t where (b < 10000 or c < 10000) and (a < 10 or d < 10) and f < 10;
 explain format="dot" select * from t where (a < 50 or b < 50) and f > 100;
+explain select /*+ use_index_merge(t, tb, tc) */ * from t where b < 50 or c < 5000000;
```

Comment:
Can you add some comments describing how do we generate the data/stats in https://github.com/pingcap/tidb/pull/12843/files#diff-a9a38bff443b96bc5181dc6bb52444e0R6?